src/version.ts

```diff
@@ -1 +1 @@
-export const version = '4.3.1';
+export const version = '4.5.0';
```

Comment:
did we skip 4.4?

Reply:
Hmm, no I think I forgot to run `yarn build` last time. The file should be ok in the published packages because there we run `yarn build` prepublish

Reply:
ok!